services/core-api/app/api/mines/permits/permit/resources/permit.py

```diff
@@ -249,10 +296,37 @@ def put(self, permit_guid, mine_guid):
         if not permit:
             raise NotFound('Permit not found.')
 
+        json_data = request.json
+
+        permit_prefix = permit.permit_no[0]
+        is_exploration = permit.permit_no[1] == "X" or permit.is_exploration
```

Comment:
Might be good to bake the logic to determine whether a permit is exploration or not into a hybrid property as well.

Reply:
discussed on a call and decided to keep as it is